tests/suite/js.html

```diff
@@ -456,8 +456,7 @@
     bar = /\/{/
     baz = //{//
 
-    // The following is known to be broken
+    // The following is no longer broken \o/
     /function() {/
```

Comment:
This parses, barely. Ignoring all the comments, we have

```
baz = /function() {/
```

i.e.: `baz` is assigned the value of a regex.

if above you'd have introduced another variable above, say `quux = 5`, it'd read

```
quux = 5

// The following...
/function () {/
```

would be parsed as

```
quux = 5 / function () {/
```

and the first `/` would thus be a division, while the second `/` would be the start of an unterminated regular expression.

Reply:
Luckily, I don't care whether it parses, I only care whether it indents!